### ISSUE TITLE
Why have we lost the deprecation status?

### ISSUE THREAD

**Issue by softdays** [SonarQubeCommunity/sonar-findbugs#79]
Please could you explain me why since 3.4 we have lost the deprecation status?

**Comment by h3xstream** [comment]
The deprecation was not representing the deprecation of those FindBugs rules. **It was a indicator that those rules where having "equivalent" version in Sonar-Quid engine.**
In remains that on short term, sonar-quid will not on short terms have the coverage that FindBugs has. The re-implementation of many of the rules were partial. My goal is not to say FindBugs is "better" but it will not be replace.

Historically, their was a [announce that FindBugs will soon be remove from SonarQube](https://blog.sonarsource.com/sonarqube-java-analyzer-the-only-rule-engine-you-need/).

> Our goal is that at some point in 2015 we’ll stop shipping the FindBugs plugin by default with the platform (we’ll still support it and provide it through the update center) because out of the box, the Java Plugin will provide at least as much (hopefully more!) value as FindBugs.

In practice, the default engine of Sonar is noisy and does not have a fraction of the rules. (370 rules vs over 800) Sonar team did not like the maintenance of the plugin vs what it brings.

In conclusion, Sonar-Quib and FindBugs will have their on strength. Some people will prefer either. The deprecation flag was simply a distraction for the FindBugs users.

**Comment by softdays** [comment]
Thanks for your feedback. I was aware of the SonarQube deprecation strategy regarding checkstyle, pmd and findbugs. Our compagny was using the deprecation status to follow the replacement of the rules. Now we have to find an other way to do this...
